ASP-NET/ASP-NET/Services/ProductService.cs

```diff
@@ -0,0 +1,79 @@
+﻿using ASP_NET.Context;
+using ASP_NET.Dto;
+using ASP_NET.Entities;
+using Microsoft.EntityFrameworkCore;
+using System.Text.Json;
+
+namespace ASP_NET.Services;
+
+public class ProductService : IProductService
+{
+    private readonly ApplicationDbContext _context;
+
+    private const string PublicProductsPhotoFolder = "public/products";
+
+    public ProductService(ApplicationDbContext context)
+    {
+        this._context = context;
+    }
+
+    public async Task<ResultResponseDto> CreateProductAsync(CreateProductDto body)
+    {
+
+        if (body.Photo == null)
+        {
+            return new ResultResponseDto { Message = "Photo is required", Success = false };
+        }
+
+        var uploadsFolder = Path.Combine(PublicProductsPhotoFolder);
```

Comment:
I would move logic for storing files to separate class IFileManager for better reusability and testability